docs/feature-guides/clarinet-js-sdk.md

```diff
@@ -165,8 +163,7 @@ const counter = simnet.getDataVar("counter", "count");
 getMapEntry(contract: string, mapName: string, mapKey: ClarityValue): ClarityValue
 ```
 
-Get the value of a map entry by its key.  
-Note that it will always return an optional value (`(some <value>)` or `none`). Just like Clarity `map-get?`.
+Get the value of a map entry by its key. Note that it will always return an optional value (`(some <value>)` or `none`). Just like Clarity `map-get?`.
```

Comment:
```suggestion
Get the value of a map entry by its key.
Note that it will always return an optional value (`(some <value>)` or `none`). Just like Clarity `map-get?`.
```